pkg/network/ovn_kubernetes.go

```diff
@@ -141,6 +141,7 @@ func renderOVNKubernetes(conf *operv1.NetworkSpec, bootstrapResult *bootstrap.Bo
 	if err != nil {
 		return nil, errors.Wrap(err, "failed to render manifests")
 	}
+	objs = append(objs, manifests...)
```

Comment:
ic, so the things to replace are in manifests, not in objs

Reply:
yep, manifests contain the rendered ovn-k objects

Reply:
I think that the readibility will improve if we s/manifests/renderedObjs/ , but that is only a comment, not asking for changing